Patir 3 fixes (#12409)
endless-sky/endless-sky: data/kahet/kahet missions.txt

Reviewer: ```suggestion
	description "Accompany the Remnant team and their cargo to <system> exactly on <date>, making sure nobody is left behind on the jump in."
```
While we're here, I might suggest this change to the description, or at least something similar.

Author: Added manually

Reviewer 2: ```suggestion
			`	"I would come with you if I could," he signs with trepidation, "but I am required here at the moment, and it is a task that can be easily done by a single person. Follow the procedure Dusk showed you, and you will be done in no time."`
		branch "Voice From The Past 0 fix"
			not "builder signal"
			action fail "Ka'het: Voice From The Past 0"
```
Would this work?

Author: Not like that, but I have added a version that should work manually

		not "ka'het: beyond Patir"
	on offer
		event "ssil vida changes"

> 		fail "Ka'het: Voice From The Past 0"

Reviewer 2: ```suggestion
		if not "builder signal"
			fail "Ka'het: Voice From The Past 0"
```
So VFTP0 only fails at this point if the player hasn't yet interacted with the satellite. If they have already, it should be fine at this point because there will be visits to a primary Remnant world and then Ssil Vida before PM28, so VFTP2 can be completed because the player won't be locked in the pocket dimension.

Author: Is "if" actual mission sintax? I've never seen it before, and I can't find it on the wiki

Reviewer 2: Uhm... probably not then.

Reviewer 2: How about adding it on to the conversation? If you make a branch with no text in it but just the condition at the end of the conversation, wouldn't that work?

		event "Patir moves" 6
		event "Patir returns" 20
	on visit

> 		dialog "You've arrived at <planet>, but either your escort, your passengers or your cargo have been left behind in the Milky Way!"

Reviewer: ```suggestion
		dialog "Your escort, your passengers or your cargo have been left behind in the Milky Way, and the mission is a failure. If you wish to continue Patir content, you may wish to reload."
```
If the mission is a guaranteed failure at this point, this should probably be written more like the "stuck in Patir" dialogs, since there's no way to get back in time.

Though, I would wonder if this should be an `on fail` dialog instead of an `on visit` message, and just delete it as an `on visit` message since there's no purpose warning the player it's blocked if it's incompletable.

Author: On fail would take one more day, right? If that is the case, I feel like on visit works better

			dialog
				`You make a close pass above this lifeless satellite, abandoned in a graveyard orbit for who knows how long. All of a sudden, your commlink emits a weak, garbled, whistle-like sound, as if receiving a communication from the Builder spacecraft. While you can't understand it, it might be worth paying another visit to the Remnant.`
			set "builder signal"
		on kill
			dialog
				`A glance at the radar screen reveals that the nearby Builder satellite exploded, presumably due to the damage incurred. A wasted opportunity.`
			clear "builder signal"

mission "Ka'het: Voice From The Past 0 Patir 3"
	landing
	invisible
	non-blocking
	to offer

> 		has "Ka'het: Patir Mystery 35: done"

Reviewer 2: ```suggestion
		has "Ka'het: Patir Mystery 35: done"
		not "builder signal"
```
The only place that I could find the `builder signal` being cleared was if the player kills the satellite, so other than that case, this would prevent duplication if the player had already interacted with it before PM14.
